README.md

```diff
@@ -1,3 +1,59 @@
 # Kinto React Boilerplate
 
 A sample [Kinto](https://github.com/mozilla-services/kinto.js) & [React](http://facebook.github.io/react/) project boilerplate.
+
+## Scope
+
+The main purpose of this boilerplate is to speed up the conception of a Kinto JavaScript application, with basic setup for unit and functional testing. It comes with:
+
+* [React + ES6](https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html) classes, using [Babel](https://babeljs.io);
+* Setup **test environment**, using [Mocha](http://mochajs.org/)+[Chai](http://chaijs.com/), [Sinon](http://sinonjs.org/) and [Karma](karma-runner.github.io/) for React components;
+* Setup **build and packaging**, using [webpack](http://webpack.github.io/);
+* Basic asynchronous store for collection records <sup>[1](#note-flux)</sup>;
+* Fully tested demo CRUD app, with sync button.
+
+Beyond this scope, we provide [some recipes](https://github.com/Kinto/kinto-react-boilerplate/labels/Tutorial) to help you going further !
+
+><a name="note-flux">1</a>: *Replacing it with a Flux library is up to you :)*
+
+## Getting started
+
+### Install
+
+Start a new project, and fetch the boilerplate:
+
+    $ mkdir newproject && cd $_
+    $ git init .
+    $ git remote add boilerplate https://github.com/Kinto/kinto-react-boilerplate.git
+    $ git merge boilerplate/master
+
+Install environment and run:
+
+    $ npm install
+    $ npm start
+
+Demo application is available at http://localhost:3000 ! With auto-refresh when code changes !
```

Comment:
There are no spaces between the end of a sentence and a punctuation mark in English. Also, URLs are a funny thing to cram into actual sentences - especially when they come at the end since it makes for awkward orthography. Consider either splitting this on to two lines or making the URL a link (that way it's visually separate).